cli/smartcontract/smart_contract.go

```diff
@@ -287,7 +287,10 @@ func NewCommands() []cli.Command {
     * 'NNQk4QXsxvsrr3GSozoWBUxEmfag7B6hz5'
     * 'NVquyZHoPirw6zAEPvY1ZezxM493zMWQqs:Global'
     * '0x0000000009070e030d0f0e020d0c06050e030c02'
-    * '0000000009070e030d0f0e020d0c06050e030c02:CalledByEntry,CustomGroups'   
+    * '0000000009070e030d0f0e020d0c06050e030c02:CalledByEntry,` +
+					`CustomGroups:0206d7495ceb34c197093b5fc1cccf1996ada05e69ef67e765462a7f5d88ee14d0'
```

Comment:
We need to specify exactly how to pass contracts/groups above where `CustomContracts` and `CustomGroups` scopes are defined.